x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/utils.ts

```diff
@@ -160,6 +160,25 @@ export const getVulnMgmtCloudFormationDefaultValue = (packageInfo: PackageInfo):
   return cloudFormationTemplate;
 };
 
+export const getCspmCloudFormationDefaultValue = (packageInfo: PackageInfo): string => {
+  if (!packageInfo.policy_templates) return '';
```

Comment:
might be better to return undefined instead of empty string.
typing the return as `string | undefined` allows the consumer of the function to know it can fail and check if he got `undefined` instead of the URL. but with an empty string you need "inside knowledge" to know that you need to check for that.